Makefile

```
@@ -70,26 +75,23 @@ gen-files:
 	$(MAKE) $(GENERATED_FILES)
 
 $(BINDIR)/deepcopy-gen: vendor
-	$(DOCKER_GO_BUILD) \
-		sh -c 'go build -o $@ $(LIBCALICO-GO_PKG)/vendor/k8s.io/code-generator/cmd/deepcopy-gen'
+	$(DOCKER_GO_BUILD) sh -c 'go build -mod=vendor -o $@ $(LIBCALICO-GO_PKG)/vendor/k8s.io/code-generator/cmd/deepcopy-gen'
 
 ./lib/upgrade/migrator/clients/v1/k8s/custom/zz_generated.deepcopy.go: $(UPGRADE_SRCS) $(BINDIR)/deepcopy-gen
-	$(DOCKER_GO_BUILD) \
-		sh -c '$(BINDIR)/deepcopy-gen \
-			--v 1 --logtostderr \
-			--go-header-file "./docs/boilerplate.go.txt" \
-			--input-dirs "$(LIBCALICO-GO_PKG)/lib/upgrade/migrator/clients/v1/k8s/custom" \
-			--bounding-dirs "github.com/projectcalico/libcalico-go" \
-			--output-file-base zz_generated.deepcopy'
+	$(DOCKER_GO_BUILD) sh -c '$(BINDIR)/deepcopy-gen \
+		--v 1 --logtostderr \
+		--go-header-file "./docs/boilerplate.go.txt" \
+		--input-dirs "$(LIBCALICO-GO_PKG)/lib/upgrade/migrator/clients/v1/k8s/custom" \
```

This target is still broken

```suggestion
		--input-dirs "./lib/upgrade/migrator/clients/v1/k8s/custom" \
```

---

Fixed